src/components/Tabs/Tabs.test.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import Tabs from './index';
+
+const TestTabs = [
```

Comment:
Also, create a `baseProps` or `props` object here and use it for props.

That way, if we decide to change props later, we can do it in one place.

Check MarketingCard.test.tsx or Layout.test.tsx, for reference.